### PR TITLE
Fix script for deploy

### LINKED ISSUE #14: Configure CD using GitHub actions
Implement CD pipeline using GitHub actions: on push events it uploads configuration files to server and runs docker-compose commands via ssh to restart the deployment.

### PR DESCRIPTION
**Pull request by genryxy**
Closes #14 
Configured CD pipeline using actions. It copies file to Artipie server using rsync via ssh and after that run script for restarting Artipie server.

### CODE REVIEW

**Comment by genryxy** [comment]
@g4s8 could you please check again. It tried to retype dash according to [this](https://stackoverflow.com/questions/66321206/error-docker-context-create-requires-exactly-1-argument)

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -2,25 +2,25 @@ name: "Deploy Artipie"
...
+          rsync -r /home/runner/myw/_work/central/central/* ${{ secrets.REMOTE_HOST }}:${{ secrets.REMOTE_TARGET }} -v
+          echo "----- rsync passed successfully -----"
+          docker context rm remote -f
+          echo "----- remove context 'remote' -----"
+          docker context create remote --docker "host=ssh://${{ secrets.REMOTE_HOST }}"
+          echo "----- context was created -----"
           docker context use remote
-          docker-compose build
+          echo "----- used remote context -----"
+          docker-compose build --build-arg https_proxy=http://localhost:3128
```
Comment (g4s8) [inline .github/workflows/deploy.yml:19]: @genryxy no-no, just use step environment to setup proxy config:
```yaml
- name: Deploy to server
  env:
    http_proxy=http://localhost:3128
  run: |
    your command...
```

**Comment by genryxy** [comment]
@g4s8 could you please take a look. It started to work now

**Comment by g4s8** [comment]
@genryxy could you please edit PR description according to our contribution guid: https://github.com/artipie/artipie/blob/master/CONTRIBUTING.md#pull-request-style

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -2,25 +2,15 @@ name: "Deploy Artipie"
...
-          SSH_REMOTE_HOST: ${{ secrets.REMOTE_HOST }}
+          http_proxy: "http://localhost:3128"
         run: |
-          docker context create remote ‐‐docker "host=ssh://$SSH_REMOTE_USER"
-          docker context use remote
-          docker-compose build
-          docker-compose pull artipie
-          docker-compose down
-          docker-compose up -d
+          rsync -r /home/runner/myw/_work/central/central/* ${{ secrets.REMOTE_HOST }}:${{ secrets.REMOTE_TARGET }} -v
```
Comment (g4s8) [inline .github/workflows/deploy.yml:13]: Absolute path here is not a good idea: we can change runner or restart it from another directory, and it'll break the script. You can use relative path (relative to repository location) or use `GITHUB_WORKSPACE` environment, see: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -2,25 +2,15 @@ name: "Deploy Artipie"
...
+          echo "----- rsync passed successfully -----"
```
Comment (g4s8) [inline .github/workflows/deploy.yml:14]: No need to put debugging message here. If you want verbose output, just enable `-v` flag for `rsync` command.

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -2,25 +2,15 @@ name: "Deploy Artipie"
...
+          ssh ${{ secrets.REMOTE_HOST }} "./restart.sh"
+          echo "----- successfully finished script -----"
```
Comment (g4s8) [inline .github/workflows/deploy.yml:16]: Same here.

**Comment by genryxy** [comment]
@g4s8 edited description and title

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy please check my comments

**Comment by genryxy** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Pull request merged**